lib/backends/sdc/build.js

```diff
@@ -891,8 +894,266 @@ function runBuildCommand(opts, callback) {
 }
 
 
+/**
+ * Commit (create) an image from a given container.
+ *
+ * @param {Object} opts
+ * @param {String} opts.author - Author name for the image
+ * @param {Array} opts.changes - Docker image changes to make.
+ * @param {String} opts.comment - Comment notes for the image.
+ * @param {Object} opts.config - Image config object
+ * @param {Boolean} opts.pause - common.boolFromQueryParam(req.query.pause),
+ * @param {Object} opts.req - Request instance.
```

Comment:
Just as the first argument - will update that comment. Note: opts in this case should be what was supplied via the docker client (i.e. query params).